activitysim/defaults/tables/skims.py

```diff
@@ -39,3 +39,17 @@ def sovmd_skim(nonmotskm_matrix):
 def sovpm_skim(nonmotskm_matrix):
     # FIXME use the right omx file
     return skim.Skim(nonmotskm_matrix, offset=-1)
+
+
+@sim.injectable(cache=True)
+def skims():
+    skims = skim.Skims()
+    # FIXME - this is reusing the same skim as all the different kinds of skims
+    for typ in ["SOV_TIME", "SOVTOLL_TIME", "HOV2_TIME",
```

Comment:
I think the main thing is that we don't want to force TOUR and TRIP skims to be the same...even though they might be the same variables, they aren't the same files.